src/Core/Prefab/GlobeView.js

```diff
@@ -193,7 +193,7 @@ function GlobeView(viewerDiv, coordCarto, options = {}) {
     const sun = new THREE.DirectionalLight();
     sun.position.set(-0.5, 0, 1);
     sun.updateMatrixWorld(true);
-    this.scene.add(sun);
+    wgs84TileLayer.object3d.add(sun);
```

Comment:
For sun I don't know, but the atmosphere is linked to the ellipsoid right? It *must* be added to the same root object as the WGS84 layer imo.